verilog/tools/ls/vscode/package.json

```diff
@@ -15,14 +15,14 @@
   "contributes": {
```

Comment:
Can you also change the version above from 0.0.2700 to 0.0.2800 ? That way it is easy to determine what version is active when looking at the plugin settings.